### ISSUE TITLE
Centers selection method

### ISSUE THREAD

**Issue by cmaerzluft** [carolyner/imabc#52]
Right now the code picks the N_centers best centers based on total distance (which is calculated using only targets distances for targets still being updated). This could potentially impact the algorithms ability to efficiently calibrate if it gets stuck in a local optima.

Carolyn and I discussed alternatives. One idea we thought might be worth testing first would be to randomly select centers from the current good draws if we are having trouble updating the bounds after a set number of iterations.
We could also randomly sample the parameters once all the targets have met the stopping bounds, but in this case we would weight by a function of the sample weight.

@carolyner, does that accurately lay out what we discussed yesterday?

**Comment by carolyner** [comment]
yes, though I think this is on hold as we are trying to resolve some issues with finding the upper and lower limits of the bounds.
